### DISCUSSION TITLE
Make Picture Entity Card templatable

### DISCUSSION THREAD

**Discussion by manutenfruits** [home-assistant/frontend#8336] [Dashboard]
I want to use the URL of a sensor attribute as the `state_image` in the Picture Entity Card, if it supported templates this would be great.

Sample use cases:
* Scraper custom sensors that retrieve the URL of an image that changes over time.

**Comment by djbx83** [comment]
This would be very useful, especially when you have an image URL inside an attribute.
For example, I have a http-rest sensor that exposes an image inside a json and I'm going crazy to display this image in a dashboard.
If picture-entity was templetable, it would solve a lot of headaches...
